dataRetriever/factory/dataPoolFactory_test.go

```diff
@@ -76,43 +88,30 @@ func TestNewDataPoolFromConfig_BadConfigShouldErr(t *testing.T) {
 	args.Config.HeadersPoolConfig.MaxHeadersPerShard = 0
 	holder, err = NewDataPoolFromConfig(args)
 	require.Nil(t, holder)
-	fmt.Println(err)
 	require.True(t, errors.Is(err, headersCache.ErrInvalidHeadersCacheParameter))
 	require.True(t, strings.Contains(err.Error(), "the cache for the headers"))
 
 	args = getGoodArgs()
 	args.Config.TxBlockBodyDataPool.Capacity = 0
 	holder, err = NewDataPoolFromConfig(args)
 	require.Nil(t, holder)
-	fmt.Println(err)
 	require.NotNil(t, err)
 	require.True(t, strings.Contains(err.Error(), "must provide a positive size while creating the cache for the miniblocks"))
 
 	args = getGoodArgs()
 	args.Config.PeerBlockBodyDataPool.Capacity = 0
 	holder, err = NewDataPoolFromConfig(args)
 	require.Nil(t, holder)
-	fmt.Println(err)
 	require.NotNil(t, err)
 	require.True(t, strings.Contains(err.Error(), "must provide a positive size while creating the cache for the peer mini block body"))
 
 	args = getGoodArgs()
 	args.Config.TrieSyncStorage.Capacity = 0
 	holder, err = NewDataPoolFromConfig(args)
 	require.Nil(t, holder)
-	fmt.Println(err)
 	require.True(t, errors.Is(err, storage.ErrCacheSizeInvalid))
 	require.True(t, strings.Contains(err.Error(), "the cache for the trie nodes"))
 
-	args = getGoodArgs()
```

Comment:
Trivial test, but took quite some time (i.e. 1 minute), since it was attempting a number of retries.